### ISSUE TITLE
Where are the Prolog communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/prolog#62]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Prolog supported by one or more large organizations?
* Does Prolog have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Prolog? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Prolog? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Prolog? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Prolog?
* Are there popular newsletters for Prolog?
* Is Prolog taught at programming bootcamps? (If so, what are some examples?)
* Is Prolog taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Prolog?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by Average-user** [comment]
I can try to answer some of the question, not many though.

- The IRC channel in freenode (##Prolog) is quite active. 
- The Prolog [subreddit](https://www.reddit.com/r/prolog/)
- I wouldn't say that there are many places dedicated to mentoring Prolog. But there are a few good books and websites. These are mentioned in the [exercism web page](http://exercism.io/languages/prolog/learning)

**Comment by kytrinyx** [comment]
There's a twitter account that is helpful for keeping up to date: https://twitter.com/SWI_Prolog

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼
